portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -16,37 +16,43 @@
 
 import java.io.IOException;
 import java.util.*;
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.PreparedQuery;
-import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.*;
```

Comment:
FYI: Most Java style guides discourage using `.*` imports because they end up cluttering the global scope, especially if you import things like `java.util.*`. If you're using an IDE like IntelliJ, it should handle explicitly listing the imported names for you, but otherwise I understand if you want to avoid the extra typing.